docs/guides/rookieguide.md

```diff
@@ -1,7 +1,126 @@
+
+Q: Na stronie z modlistą świeci się na żółto jedna z paczek. Co to znaczy?
+
+A: Że ta żółta paczka jest obecnie załadowana na serwerze. Nie uda ci się dołączyć do serwera odpalając grę z inną paczką modów, niż jest obecnie załadowana. Gra będzie sypać błędami. Paczka musi być aktualna, więc najlepiej parę godzin przed misją pobrać paczkę jeszcze raz, żeby mieć pewność, że wszystko się zaktualizowało. Raz na ruski rok mission maker zmieni zawartość paczki na chwilę przed misją (np. godzinę), wtedy jest dawany cynk na discordzie. Ale ta sytuacja dzieje się dość rzadko.
+
+Q: Bardzo dużo teorii, a ja nie potrafię szybko czytać. Czy mogę jakoś w praktyce wypróbowac zaprezentowaną tu wiedzę?
+
+A: Tak, serwer jest z reguły odpalony 24/7, po uzyskaniu rangi rekruta możesz normalnie popróbować wejść na serwer, jeżeli jakiś mission maker nie testuje misji, to odpalony jest default modlist (ale jak zawsze odsyłam do sprawdzenia “co się świeci” na stronie z  modlistami) na mapie Altis - można sobie pojeździć i polatać, poleczyć boty. Są spawnery tu i tam.
+Zachęcam również do sprawdzenia działania ACRE/TeamSpeaka, na serwer TS też możecie wchodzić bez strachu, czujcie się jak u siebie. ACRE najlepiej przetestować z drugą osobą, poproście rekruterów lub innych rekrutów, bo może też szukają partnera/ki do testów.
+
+Q: Ile przeciętnie trwa misja?
+
+A: Od 1,5h do 2,5h. Jeżeli coś się wysypie - może trwać dłużej. Misje, które są JointOps z innymi grupami zazwyczaj trwają dłużej, czasem nawet do 4 godzin.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+Q: Nie jestem już rekrutem, zostałem przyjęty, chcę się zapisać na misję!
+
+A: Mission maker utworzy jakiś czas przed misją (z reguły kilka dni) tzw. zapisy na misję na naszym Discordzie. Bot (Boderator) zawoła wtedy cały serwer, więc na pewno nie przeoczysz. Każda misja ma utworzony dedykowany kanał z zapisami, gdzie zawarty jest briefing, data misji, data zamknięcia zapisów, poszczególne sekcje i role do obsadzenia. Zapisy odbywają się poprzez kliknięcie adekwatnej reakcji po daną sekcją. Reakcję klikamy raz (czasami może być lag), możemy się oczywiście wypisać przed upływem terminu końca zapisów, później pod groźbą bana na zapisy jesteśmy zobligowani stawić się na misji.
+
+![Przykładowe zapisy](../_data/guides/rookie/rookie4.png)
+
+**Sekcja “Delta” i wykaz zajętych/wolnych ról. Obsadzony jest np. dowódca drużyny, ale wolne są sloty LKMisty, anti-tank, granadiera. Nasza reakcja (zapis na misję) zostanie odrzucona w przypadku próby zapisania się na dowódcę lub medyka (bo są zajęte), ale każda inna rola wciąż ma jeden “slot” wolny. W czerwonym polu klikamy ikonę interesującej nas roli.**
+
+**Ten tutorial stanowi nowszą wersję trochę bardziej śmieszkowego tutoriala napisanego przez jednego z rekruterów - Nakimetelesa. Tamta wersja posiada kilka przestarzałych informacji, lecz polecam zapoznać się z nią: https://docs.google.com/document/d/1nOPqnTeu9Flph8oBtN2FFMQ5dUvwFet_Y80uH1ckp1s/edit**
```

Comment:
I'd not say we recommend viewing old documents. Let's leave them _for reference_, nothing else.
```suggestion
**Ten tutorial stanowi nowszą wersję trochę bardziej śmieszkowego tutoriala napisanego przez jednego z rekruterów - Nakimetelesa. Tamta wersja posiada kilka przestarzałych informacji, lecz wciąż jest dostępna jeżeli istnieje taka potrzeba: https://docs.google.com/document/d/1nOPqnTeu9Flph8oBtN2FFMQ5dUvwFet_Y80uH1ckp1s/edit**
```